packages/ui-tests/cypress/support/next-commands/datamapper.ts

```diff
@@ -1,13 +1,13 @@
 Cypress.Commands.add('attachSourceBodySchema', (filePath: string) => {
   cy.get('[data-testid="attach-schema-sourceBody-Body-button"]').click();
   cy.get('[data-testid="attach-schema-file-input"]').attachFile(filePath);
-  cy.get('[data-testid="expand-source-icon"]').should('be.visible');
+  cy.get('[data-testid^="expand-source-icon"]').should('be.visible');
```

Comment:
Does this still work if a schema is already attached to some parameter, say `Account` - in that case, DOM would have multiple ones, such as `expand-srouce-icon-Account` and `expand-source-icon-sourceBody`